week-1-ts-ecosystem/w01e01-typescript-basics/src/core/enviroment.ts

```diff
@@ -0,0 +1 @@
+export const ADULT_AGE = 18;
```

Comment:
Bardzo fajnie @zajewski  że wychwyciłeś tutaj Magic Number. i dałeś mu osobne miejsce.
Dyskusyjne jest czy to zmienna środowiskowa - w mojej opinii - no na siłę, można uznać że z jakichś powodów chcemy to konfigurować z zewnątrz, jednak - nawet jeśli to musiałby być w `.env`

Myślę finalnie że - mimo wszystko dałbym to jako `constants` jeśli już to jakoś "katoegoryzować"
albo po prostu niech nie będzie magic ale mieszka razem z `age.utils`

Reply:
chodziło mi o to że w różnych krajach może być różny adult age aczkolwiek faktycznie może niekoniecznie zmienna środowiskowa